library/network/src/modules/PortRanges.rb

```diff
@@ -586,7 +586,7 @@ def FlattenServices(old_list, protocol)
               new_max = this_max_pr
               # the second one partly covers the first one (by its left side)
             elsif Ops.greater_or_equal(min_pr, this_min_pr) &&
-                Ops.less_or_equal(max_pr, this_max_pr)
+                Ops.greater_or_equal(max_pr, this_max_pr)
```

Comment:
Nice catch.

Reply:
yeap, rubocop complain that condition is identical and then I spend 5 minutes thinking how it should be correctly.